.circleci/config.yml

```diff
@@ -1019,6 +1019,19 @@ workflows:
           requires:
             - lint-3.7
 
+      - scenario-player-integration-test:
+          name: scenario-player-integration-test
+          py-version: "3.7"
+          requires:
+            - smoketest-matrix-production-geth-3.7
+            - smoketest-matrix-production-parity-3.7
+            - smoketest-matrix-development-geth-3.7
+            - smoketest-matrix-development-parity-3.7
+            - test-unit-3.7
+            - test-fuzz-3.7
+            - test-mocked-3.7
+            - shellcheck
```

Comment:
What would go wrong if we didn't have these deps? They are already part of `test`, after all?

Reply:
I think everything would run at the same time

Reply:
The requirements could probably factored out into a target that is used by `test-integration-matrix-geth-3.7`, `test-integration-matrix-parity-3.7` and `scenario-player-integration-test`. But no need to do that now.

It's great that you took the time to fix this, as the SP-smoketest really is quite slow.